apps/server/src/modules/files/entity/file.entity.spec.ts

```diff
@@ -94,6 +94,41 @@ describe(FileEntity.name, () => {
 		});
 	});
 
+	describe('removCreatorId', () => {
+		describe('when called on a file that contains matching creatorId', () => {
+			it('should properly remove this creatorId', () => {
+				const file = fileEntityFactory.build({
+					ownerId: mainUserId,
+					creatorId: mainUserId,
+				});
+
+				const expectedFile = copyFile(file);
+				expectedFile._creatorId = undefined;
+
+				file.removeCreatorId(mainUserId);
+
+				expect(file).toEqual(expectedFile);
+			});
+		});
+
+		describe("when called on a file that doesn't have any permission with given refId", () => {
+			it('should not modify the file in any way (including the other present permissions)', () => {
+				const file = fileEntityFactory.build({
+					ownerId: mainUserId,
+					creatorId: mainUserId,
+				});
+
+				const originalFile = copyFile(file);
+
+				const randomUserId = new ObjectId().toHexString();
```

Comment:
Please move it to the `setup()` function.

Reply:
done